cumulusci/core/config/BaseProjectConfig.py

```diff
@@ -389,19 +389,23 @@ def get_github_api(self):  # pragma: nocover
         gh = get_github_api(github_config.username, github_config.password)
         return gh
 
-    def get_latest_version(self, beta=False):
+    def get_latest_version(self, beta=False, tag=False):
```

Comment:
Using an arg to control what kind of data is returned is a bit of a code smell. I'd rather see two functions, `get_latest_release_version` and `get_latest_release_tag`, and factor their common logic into a `get_latest_release`

Reply:
Fixed in latest commit